src/esp/physics/bullet/BulletArticulatedObject.cpp

```diff
@@ -440,6 +440,26 @@ std::vector<float> BulletArticulatedObject::getJointPositions() {
   return positions;
 }
 
+std::vector<float> BulletArticulatedObject::getJointMotorTorques(double fixedTimeStep) {
+  std::vector<float> torques(btMultiBody_->getNumLinks());
```

Comment:
Don't you want to separately call reserve on this? This will allocate 2x getNumLinks won't it?